script.js

```diff
@@ -10,8 +12,169 @@ hamburger.addEventListener('click', () => {
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent.',
+    image: 'worksimg/Snapshoot Portfolio@3x.svg',
+    technologies: ['html', 'ruby', 'css', 'javaScript', 'bootstrap', 'gitHub'],
+    liveVersion: '#',
+    source: '#',
+  },
+
+  {
+    id: 2,
+    name: 'Multi-Post Stories',
+    titleDescription: ['FACEBOOK', 'Full Stack Dev', '2015'],
+    mainDescription: 'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent.',
+    image: 'worksimg/Snapshoot Portfolio@2x.svg',
+    technologies: ['html', 'ruby', 'css', 'javaScript', 'bootstrap', 'gitHub'],
+    liveVersion: '#',
+    source: '#',
+  },
+
+  {
+    id: 3,
+    name: 'Tonic',
+    titleDescription: ['CANOPY', 'Back End Dev', '2015'],
+    mainDescription: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent.',
+    image: 'worksimg/Snapshoot Portfolio@1x.svg',
+    technologies: ['html', 'css', 'javaScript', 'bootstrap'],
+    liveVersion: '#',
+    source: '#',
+  },
+];
+
+const displayModal = (id) => {
+  const displayIdOf = cardsContainer.findIndex((card) => card.id === +id);
+  worksContainer.style.display = 'flex';
+  overlay.style.display = 'block';
+  let html = '';
+
+  html += `<div class= "part1">
+    <div class="main1">
+        <h2 id="six">${cardsContainer[displayIdOf].name}</h2>
+        <button class="close">
+          <i class="las la-times"></i>
+        </button>
+    </div>
+        <ul class="frame">
+          <li id="client">${cardsContainer[displayIdOf].titleDescription[0]}</li>
+          <li id="dot"></li>
+          <li id="role">${cardsContainer[displayIdOf].titleDescription[1]}</li>
+          <li id="dot"></li>
+          <li id="year">${cardsContainer[displayIdOf].titleDescription[2]}</li>
+      </ul>
+      <img class="image" src="${cardsContainer[displayIdOf].image}">
```

Comment:
- [ ] Good job automatically rendering the project with javascript as required 👏 . However on mobile, when the pop-up comes up a user has to scroll down to see the details which have the links below. for better web accessibility and a great user experience, kindly make the render fit into the screen just as it is on the figma design.. this way, you give a user a good experience, and stick to best practices 👏 
- You can check for overflows in your CSS concerning this 

## Your view:
![image](https://user-images.githubusercontent.com/50990925/217966792-c8af4f05-cc75-4ef1-bf37-f8d5ed61696c.png)

## Figma:
![image](https://user-images.githubusercontent.com/50990925/217966869-ad39f832-3f2f-490a-9593-87990106e15e.png)